src/app/components/editor/UrlInput.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { setQueryParam } from '@/lib/utils/setQueryParam';
+import { useLocale } from '@/locales/useLocale';
+import { Button } from '@nextui-org/button';
+import { Input } from '@nextui-org/input';
+import { usePathname, useRouter, useSearchParams } from 'next/navigation';
+
+import { useState } from 'react';
+
+interface UrlInputProps {
+  urlOverwrite?: string | undefined;
+}
+
+export function UrlInput({ urlOverwrite = '' }: UrlInputProps) {
+  const [url, setUrl] = useState(urlOverwrite || '');
+  const searchParams = useSearchParams();
+  const pathname = usePathname();
+  const router = useRouter();
+  const locale = useLocale();
+
+  const setNewUrl = (url: string) => {
+    const current = new URLSearchParams(Array.from(searchParams.entries()));
+
+    setQueryParam(current, 'url', url);
+
+    const search = current.toString();
+    const query = search ? `?${search}` : '';
+
+    router.push(`${pathname}${query}`);
+  };
+
+  return (
+    <div className="flex items-center w-full">
+      <Input
+        type="url"
+        label="URL"
+        labelPlacement={'outside'}
```

Comment:
```suggestion
        labelPlacement='outside'
```

Reply:
Fixed in https://github.com/thirdmadman/graphiql-app/pull/70/commits/334821572656642ed7d0ba580be083ae7cdd3ab7